external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if(${ENABLE_PCMSolver})
 
         ExternalProject_Add(pcmsolver_external
             GIT_REPOSITORY https://github.com/PCMSolver/pcmsolver
-            GIT_TAG v1.1.9
+            GIT_TAG v1.1.10
```

Comment:
One more spot for 3 total, https://github.com/psi4/psi4/blob/master/psi4/CMakeLists.txt#L109